tests/snuba/api/endpoints/test_organization_events_stats.py

```diff
@@ -1024,3 +1024,32 @@ def test_top_events_with_int(self):
             [{"count": 3}],
             [{"count": 0}],
         ]
+
+    def test_top_events_with_user(self):
+        with self.feature("organizations:discover-basic"):
+            response = self.client.get(
+                self.url,
+                data={
+                    "start": iso_format(self.day_ago),
+                    "end": iso_format(self.day_ago + timedelta(hours=1, minutes=59)),
+                    "interval": "1h",
+                    "yAxis": "count()",
+                    "orderby": ["-count()"],
+                    "field": ["user", "count()"],
+                    "topEvents": 5,
+                },
+                format="json",
+            )
+
+        data = response.data
+        assert response.status_code == 200, response.content
+        assert len(data) == 5
+
+        for key, result in six.iteritems(data):
+            # Because user represents all the possible user fields eg. user.id
+            # or user.email, the key is email followed by None several times
+            if key.startswith("bar@example.com"):
```

Comment:
@markstory did some work to only return the email to the frontend for user when using the "user" alias. I think that might be beneficial here as well.